### ISSUE TITLE
Adding a "tick box" feature for pipsolar option

### ISSUE THREAD

**Issue by ouinouin** [benzino77/tasmocompiler#396]
Hi , the macro is : 
#define USE_PIPSOLAR

see : https://github.com/arendst/Tasmota/pull/20408/commits/3125181c6e56939652f8390485c3235020ac7ddc
I guess this option will gain traction amongst diyers having solar inverters. I m actually testing #define USE_PIPSOLAR with tasmocompiler and an ESP32-S3 to see if it compiles and its a success , i got no unmet dependencies or strange thing during compilation.

**Comment by benzino77** [comment]
Hi,
Thanks for pointing that out. I will leave this issue open to see if there are more people interesting in that feature to be available on the features list (I don't want to overload GUI with too many options).

Based on the information that there is only one `define` needed, one could just put it in the "Custom parameters" field (which I think is the way you tested it).

**Comment by ouinouin** [comment]
good point, its a nice way to see if there is interrest, and maybe you re right too much tick box would maybe kill the tick box :-)
